tests/TikaTextExtractorTest.php

```diff
@@ -8,8 +8,6 @@
 
 /**
  * Tests the {@see TikaTextExtractor} class
- *
- * @group tika-tests
```

Comment:
This removes a warning in when running php unit tests https://github.com/silverstripe/silverstripe-textextraction/actions/runs/19721167001/job/56503874183#step:12:103